admin/layout_controller.php

```diff
@@ -29,7 +30,22 @@
   }
 
   $dir_check= $directory_array;
+// Check all existing boxes are in the current template /sideboxes/shared
+if ($dir = @dir($boxes_directory_shared)) {
+  while ($file = $dir->read()) {
+    if (!is_dir($boxes_directory_shared . $file)) {
+      if (!in_array($file, $dir_check, TRUE)) {
+        if (preg_match('~^[^\._].*\.php$~i', $file) > 0) {
+          $directory_array[] = $file;
+        }
+      }
+    }
+  }
+  $dir->close();
+}
+sort($directory_array);

```

Comment:
I'd like to see these lines 33-48 indented 2 spaces to match the surrounding lines.